doc/nrf/libraries/modem/ntn.rst

```diff
@@ -0,0 +1,98 @@
+.. _lib_ntn:
+
+NTN
+###
+
+.. contents::
+   :local:
+   :depth: 2
+
+The NTN (Non-Terrestrial Network) library provides helper functionality for NTN operations.
+
+.. note::
+   This library is only supported by the following modem firmware:
+   - mfw_nrf9151-ntn
+
+Overview
+********
+
+The NTN library currently provides location update functionality for NTN operations.
+The library monitors location update requests from the modem and provides two methods for supplying location information, periodic updates from external GNSS and direct location setting.
```

Comment:
We talk about external GNSS and since we have internal GNSS, I think we need to mention that somewhere.